SecurityManager/Build/Remotion.Local.build

```diff
@@ -35,26 +35,37 @@
     <SkipSourceLinks>True</SkipSourceLinks>
     <SkipNuGet>True</SkipNuGet>
     <SkipDependDB>True</SkipDependDB>
-    <DisableDocker>True</DisableDocker>
+    <HostTestSitesInDocker>False</HostTestSitesInDocker>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <DockerImageDotNetFrameworkRuntime4_7 Condition="'$(DockerImageDotNetFrameworkRuntime4_7)' == ''">
+      Docker images are per default unavailable in local builds

```

Comment:
Since `DockerImageDotNetFrameworkRuntimeXX` cannot be empty (by design) since TestConfigurations are validated against supported values even if they are not run, we need to set them to a non-empty value in local builds as well